rest/taskana-rest-spring/pom.xml

```diff
@@ -170,6 +170,12 @@
       <version>${version.archunit}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>pro.taskana</groupId>
+      <artifactId>taskana-test-api</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>

```

Comment:
For what do you use the test-api?

Reply:
For these annotations:
import pro.taskana.testapi.security.JaasExtension;
import pro.taskana.testapi.security.WithAccessId;

Previously we got them from taskana-common-test, from which we detached by creating the new module.